advocacy_docs/repos/getting_started/get_your_token.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Get your token
+navTitle: Get your token
+description: How to get the token you need to download EDB software
+---
+
+## Tokens and EDB Accounts
+
+To download EDB software, you need a token. 
+This token is a string that identifies you and your eligibility to download software from EDB. 
+If you have an account on EDB, your account already has a token associated with it. 
```

Comment:
Also realising it may be wrong as I don't know if we defer token generation for all accounts which means you don't have a token initially even if you have created your account.